src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -250,7 +250,7 @@ internal override ValueTask ConnectAsync(CancellationToken cancellationToken = d
             {
                 DnsEndPoint dnsEp => (dnsEp.Host, dnsEp.Port),
                 IPEndPoint ipEp => (ipEp.Address.ToString(), ipEp.Port),
-                _ => throw new Exception($"Unsupported remote endpoint type '{_remoteEndPoint.GetType()}'.")
+                _ => throw new Exception(SR.Format(SR.net_quic_unsupported_endpoint_type, _remoteEndPoint.GetType()))
```

Comment:
@scalablecory, I assume these exceptions will change to some better type.  When they do, are the messages likely to change as well?  Wondering if we should be moving them to the resx now or what a bit until things settle.